<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQrDecoder</class>
 <widget class="QMainWindow" name="QQrDecoder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQrDecoder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,8">
    <item>
     <widget class="QPushButton" name="startDecode">
      <property name="text">
       <string>Capture And Decode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>No result</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCameraControllerWidget" name="cameraWidget" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCameraControllerWidget</class>
   <extends>QWidget</extends>
   <header>qcameracontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
